spec/controllers/replies_controller_spec.rb

```diff
@@ -1014,7 +1037,7 @@
         replies = Array.new(4) { create(:reply) }
         filtered_reply = replies.last
         get :search, params: { commit: true, post_id: filtered_reply.post_id }
-        expect(assigns(:search_results)).to match_array([filtered_reply])
+        expect(assigns(:search_results)).to match_array([filtered_reply.post.written, filtered_reply])
```

Comment:
We want this, right? For searches to find the main post? This is good!

Reply:
This is the main user-facing problem with our current model, yeah!